apps/src/applab/dataBrowser/AddKeyRow.jsx

```diff
@@ -45,7 +45,7 @@ const AddKeyRow = React.createClass({
 
   render() {
     return (
-      <tr style={dataStyles.row}>
+      <tr id="addKeyValuePairRow" style={dataStyles.row}>
```

Comment:
something i've been doing is to add uitest to the classname/id when it's only being used in a uitest (so this could be something like `uitest-addKeyValuePairRow`). This way, someone reading the component doesnt need to worry about searching through CSS, etc. to understand how the component will render